docs/docs/external-volumes.md

```diff
@@ -84,7 +84,7 @@ You can specify an external volume in your Marathon app definition. [Learn more
       "instances": 1,
       "cpus": 0.1,
       "mem": 32,
-      "cmd": "/usr/bin/tail -f /dev/null",
+      "cmd": "mkdir my-test-vol && touch my-test-vol/test.txt && ls my-test-vol/",
```

Comment:
That last time was an EE cluster and just cutting-and-pasting the first example and your suggested `cmd`. But I didn't expect the external volumes part to work, since I didn't do any of the necessary setup (the custom installation with `genconf/rexray.yaml` and so on).